packages/vue-renderer/src/renderers/ssr.js

```diff
@@ -180,7 +180,11 @@ export default class SSRRenderer extends BaseRenderer {
     if (renderContext.staticAssetsBase) {
       const preloadScripts = []
       renderContext.staticAssets = []
-      const { staticAssetsBase, url, nuxt, staticAssets } = renderContext
+      const { staticAssetsBase, nuxt, staticAssets } = renderContext
+      let url = new $URL(renderContext.url).pathname
```

Comment:
We can instead use lighter `parseURL().pathname` utility (also if prop is of path type, variable may be named pathname as well instead of url)